TicketsDemo.Domain/DefaultImplementations/TicketServiceWithLogger.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Data.Entities;
+using TicketsDemo.Data.Repositories;
+using TicketsDemo.Domain.Interfaces;
+
+namespace TicketsDemo.Domain.NewImplementations
+{
+    public class TicketServiceWithLogger : ITicketService
```

Comment:
Here you modified original TicketsService by blending logging capabilities inside the service.
This works against the OpenClosed principle.

Instead, create TicketServiceLoggingDecorator which would be responsible for logging and which would decorate original ITicketService implementation